scripts/save.sh

```diff
@@ -246,14 +246,8 @@ dump_windows() {
 			fi
 			# window_layout is not correct for zoomed windows
 			if [[ "$window_flags" == *Z* ]]; then
```

Comment:
Can't we just remove this whole `if` block? I can't see what value does a line that reassigns `window_layout` provides?